net/vpnbypass/Makefile

```diff
@@ -5,7 +5,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=vpnbypass
 PKG_VERSION:=1.3.1
-PKG_RELEASE:=7
```

Comment:
Why do you jump from 7 to 9?

Reply:
I also offer most packages from private repo and bump release numbers there more frequently than I submit PRs for changes. In order not to have the lower version in the official repo, that means that sometimes there are jumps.

Reply:
Okay I guess.